shelley/chain-and-ledger/dependencies/non-integer/doc/iohk.sty

```diff
@@ -36,9 +36,9 @@
 \newenvironment{question}
   {\begin{bclogo}[logo=\bcquestion, couleur=orange!10, arrondi=0.2]{ QUESTION}}
   {\end{bclogo}}
-\newenvironment{todo}
-  {\begin{bclogo}[logo=\bcoutil, couleur=red!5, couleurBarre=red, arrondi=0.2]{ TODO}}
-    {\end{bclogo}}
+% \newenvironment{todo}
+%   {\begin{bclogo}[logo=\bcoutil, couleur=red!5, couleurBarre=red, arrondi=0.2]{ TODO}}
+%     {\end{bclogo}}
```

Comment:
any reason to keep the commented environment?

Reply:
Unfortunately the environment conflicts with the todonote package...